packages/fx-core/src/component/generator/templates/templateNames.ts

```diff
@@ -27,6 +27,7 @@ export enum TemplateNames {
   DashboardTab = "dashboard-tab",
   TabSSR = "non-sso-tab-ssr", // handled by SsrTabGenerator
   SsoTabSSR = "sso-tab-ssr", // handled by SsrTabGenerator
+  TabSPFx = "spfx-tab", // handled by SPFxGeneratorNew
```

Comment:
Is this template name also be added in question tree node?

Reply:
Template seems hardcoded in SPFx generator for now.
<img width="370" alt="image" src="https://github.com/user-attachments/assets/3e6a76c9-4ee2-4cce-819b-4ffd7bbec5c5" />

Reply:
I see, the SPFx generator activation is not triggered by template name yet.